lib/mini_sql/sqlite/connection.rb

```diff
@@ -68,7 +74,9 @@ def run(sql, *params)
         end
         if block_given?
           stmt = SQLite3::Statement.new(raw_connection, sql)
-          yield stmt.execute
+          result = yield stmt.execute
+          stmt.close
```

Comment:
fix bloat memory, `SQLite3::Statement` created prepared query and lived in-memory sqlite

Reply:
nice!